README.md

```diff
@@ -157,7 +171,7 @@ lib/linux/x86_64/libpv_cheetah.so \
 AUDIO_DEVICE_NAME \
 lib/common/acoustic_model.pv \
 lib/common/language_model.pv \
-resources/license/cheetah_eval_linux_public.lic
+PATH_TO_YOUR_CHEETAH_LICENSE_FILE
```

Comment:
We used ${PATH_TO_YOUR_CHEETAH_LICENSE_FILE} notation for variables in shell across documentation.